packages/vike-react-styled-components/config.ts

```diff
@@ -0,0 +1,34 @@
+export { config as default }
+
+import type { IStyleSheetManager, ServerStyleSheet } from 'styled-components'
+import type { Config } from 'vike/types'
+
+const config = {
+  name: 'vike-react-styled-components',
+  require: {
+    vike: '>=0.4.203',
+    'vike-react': '>=0.4.13',
+  },
+  onAfterRenderHtml: 'import:vike-react-styled-components/__internal/onAfterRenderHtml:onAfterRenderHtml',
+  onBeforeRenderHtml: 'import:vike-react-styled-components/__internal/onBeforeRenderHtml:onBeforeRenderHtml',
+  Wrapper: 'import:vike-react-styled-components/__internal/Wrapper:Wrapper',
+  meta: {
+    styleSheetManager: {
+      env: {
+        server: true,
+        client: false,
+      },
+    },
+  },
+} satisfies Config
+
+declare global {
+  namespace Vike {
+    interface PageContext {
+      styleSheet?: ServerStyleSheet
```

Comment:
For the Vike extension, let's call it something more unique e.g. `pageContext.styledComponentsSheet`.

Reply:
What about `serverStyleSheet` ?

Reply:
It's about styled components, so the name should mention it.